src/panel/pages/timeline/Timeline.tsx

```diff
@@ -1,39 +1,58 @@
-import React, { FC } from "react";
+import React, { FC, useMemo } from "react";
 import styled from "styled-components";
 import { useTimelineContext } from "../../context";
 import { Background } from "../../components/Background";
-import { TimelineRow, TimelinePane } from "./components";
+import { TimelineRow, TimelinePane, Tick } from "./components";
 
 export const Timeline: FC = () => {
   const {
     setContainer,
-    getTimePosition,
+    scale,
```

Comment:
Getting some type errors here

Reply:
I think the TS server is using old values - it's passing type checking now 🙏